src/irmin.mli

```diff
@@ -1796,75 +1575,625 @@ module Private: sig
     (** Private repositories. *)
     module Repo: sig
       type t
-      val create: config -> t Lwt.t
+      val v: config -> t Lwt.t
       val contents_t: t -> Contents.t
       val node_t: t -> Node.t
       val commit_t: t -> Commit.t
-      val ref_t: t -> Ref.t
+      val branch_t: t -> Branch.t
```

Comment:
My plan is to expose the Git `refs` in the Git backend only. I am not totally sure how this would work, but I have some ideas that I'd like to test.